content/hi/docs/reference/glossary/daemonset.md

```diff
@@ -0,0 +1,19 @@
+---
+title: डेमनसेट (DaemonSet)
```

Comment:
DaemonSet in aka

Reply:
fundamental ->  core-object ->  workload ? Where exactly?